tests/test_runtime_json.cpp

```
@@ -195,22 +196,41 @@ TEST(RuntimeJson, NestedObject) {
     EXPECT_EQ(lh->len, 2);
     const char **data = (const char**)lh->data;
     EXPECT_STREQ(data[0], "a");
     EXPECT_STREQ(data[1], "b");
-    // Cleanup
     for (int64_t i = 0; i < lh->len; i++) free((void*)data[i]);
     free(data);
     free(keys);
-    __ry_json_free(v);
+}
+
+TEST(RuntimeJson, ObjectKeysNullInput) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)